diff --git a/scripts/react_native_pods.rb b/scripts/react_native_pods.rb
index 4115944b2e9ce286763b024356bcd695b793b23e..1b48c76a09ccae207650169533a046218560c247 100644
--- a/scripts/react_native_pods.rb
+++ b/scripts/react_native_pods.rb
@@ -404,9 +404,9 @@ def react_native_post_install(
   SPM.apply_on_post_install(installer)
 
   if privacy_file_aggregation_enabled
-    PrivacyManifestUtils.add_aggregated_privacy_manifest(installer)
+    # PrivacyManifestUtils.add_aggregated_privacy_manifest(installer)
   else
-    PrivacyManifestUtils.add_privacy_manifest_if_needed(installer)
+    # PrivacyManifestUtils.add_privacy_manifest_if_needed(installer)
   end
 
   NewArchitectureHelper.set_clang_cxx_language_standard_if_needed(installer)
